### DISCUSSION TITLE
How to pass the item in the onPressed

### DISCUSSION THREAD

**Discussion by athompson887** [letsar/flutter_slidable#312] [Q&A]
In the example code the function doNothing is called and this works fine but I cannot see how I can access the value of the row item that was swiped/pressed or even its index as the function takes no parameters.  I have looked at the build context to see if its there but I cannot find it.

This is probably very easy but I am not seeing it.

Can someone help?

**Comment by abed-asb** [comment]
Assuming you have list of row items and you want to print the index of each row then you could do the below, 
onPressed: (BuildContext context) {
                                    print(index);
                                  },

Just delete the function doNothing and assign the void to onPressed
